### ISSUE TITLE
Questions about training model on a small dataset

### ISSUE THREAD

**Issue by Anduin9527** [House-Leo/FoundIR#11]
Hi, thank you for releasing this great work!

I’m trying to apply your pretrained model to a small-scale dataset (3K) that contains three types of single degradation (motion blur/lowlight/noise). However, the current performance is unsatisfactory, and I’d appreciate some guidance:
1. I trained this model from scratch, and after ~1000 steps of training, the L1 loss quickly plateaus around 0.05 and oscillates, with no further performance gains.😢
2. I noticed the code provides three different fusion methods during test, but the paper doesn’t seem to discuss them in detail.

FYI, I’m using 4x RTX 4090 (24GB)

**Comment by Anduin9527** [comment]
Thanks for your patient reply~
